qt/qt_common/map_widget.cpp

```diff
@@ -39,11 +39,11 @@ MapWidget::MapWidget(Framework & framework, bool isScreenshotMode, QWidget * par
   , m_contextFactory(nullptr)
 {
   setMouseTracking(true);
-  // Update widget contents each 30ms.
+  // Update widget's content frequency is 60 fps.
```

Comment:
Can we have some automatic default taken from the system for example? What's the need to force 60? Won't it consume more battery on laptops and mobile Linux devices?

Reply:
Well, it is not intended to consume more battery, here is how it works:
We have the drape core which calculates frames by signals (gestures, etc) as fast as possible, and even in Debug it is usually > 60 fps. But for some reason, we do check if we have an updated frame with 30 fps freq only, so we skip intermediate already calculated frames.

60 fps is an industry standard, what kind of system settings can we check here?